.github/workflows/ci.yml

```diff
@@ -35,6 +35,9 @@ jobs:
         aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
         aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
 
+    - name: Test
+      run: nix develop '.#rust-env' -c cargo test --locked --workspace
```

Comment:
we have a feature to enable `extra-tests` you should enable that so actually all tests are run, unless you meant to keep that disabled which seems kinda counter the point of this change?

Reply:
Actually, some of those tests are failing for reasons unrelated to this PR, so I would rather defer that to a separate PR.

Reply:
ok lets look at that later, no need to block, thanks for flagging